src/components/compound/DlSearches/DlSmartSearch/components/SuggestionsDropdown.vue

```diff
@@ -121,6 +121,16 @@ export default defineComponent({
 
         const setHighlightedIndex = (value: any) => {
             highlightedIndex.value = value
+
+            const element =
+                menu.value?.portalEl.querySelectorAll('.dl-list-item')[value]
```

Comment:
@makc-dataloop menu element could have other elements inside of it instead of just dl-list-items what would happen in that case ? 

if i understood correctly this would only work for the smartsearch menu for now

Reply:
@fadiDL yes but this is DlSmartSearch/components/SuggestionsDropdown.vue - it should not be used anywhere outside of DlSmartSearch ?

Reply:
ah wait, do you mean you want this working for any kind of menu whatsoever?

Reply:
i think they do if we add the prop no ? lets take this in private and find an appropriate solution @makc-dataloop